app/app.go

```diff
@@ -377,7 +377,6 @@ func (app *GaiaApp) InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci
 
 	// Set Historical infos in InitChain to ignore genesis params
```

Comment:
Why do even need this call? Shouldn't `x/staking` InitGenesis do everything it needs?